Binärdateien a/src/libyder.so und b/src/libyder.so sind verschieden.
Binärdateien a/src/libyder.so.1.1.1 und b/src/libyder.so.1.1.1 sind verschieden.
diff -pruN a/src/Makefile b/src/Makefile
--- a/src/Makefile	2018-01-16 22:41:22.000000000 +0100
+++ b/src/Makefile	2018-01-17 12:50:56.030845423 +0100
@@ -18,8 +18,7 @@
 # You should have received a copy of the GNU General Public
 # License along with this library.	If not, see <http://www.gnu.org/licenses/>.
 #
-PREFIX=/usr/local
-CC=gcc
+PREFIX=/usr
 CFLAGS=-c -fPIC -Wall -D_REENTRANT $(ADDITIONALFLAGS)
 LIBS=-lc -lorcania
 OUTPUT=libyder.so
@@ -43,7 +42,6 @@ clean:
 install: all
 	cp $(OUTPUT).$(VERSION) $(PREFIX)/lib
 	cp yder.h $(PREFIX)/include
-	-ldconfig
 
 static-install: static
 	cp libyder.a $(PREFIX)/lib
Binärdateien a/src/yder.o und b/src/yder.o sind verschieden.
